src/shutdown.cpp

```diff
@@ -28,6 +31,7 @@ bool AbortNode(const std::string& strMessage, bilingual_str user_message)
         user_message = _("A fatal internal error occurred, see debug.log for details");
     }
     InitError(user_message);
+    Assert(g_exit_status)->store(EXIT_FAILURE);
```

Comment:
Does this make `InitShutdownState()` required before calling `AbortNode()` ? 
I suppose that could be added to the comment here. I'm just making sure I understand, I don't think you need to retouch:

https://github.com/bitcoin/bitcoin/blob/6f5f37eefd425faabd9a97a3c3028395c34f08c4/src/shutdown.h#L14-L17